src/tracks/BBTrack.cpp

```diff
@@ -370,7 +286,6 @@ void BBTCOView::update()
 }
 
 
-
 QColor * BBTrack::s_lastTCOColor = NULL;
```

Comment:
Do you think I should remove this? I don't use this in the new color implementation, and the BB track doesn't have its own color implementation anymore.

Reply:
Sounds good to me, yes.

Reply:
Great, I removed it.